build.gradle

```diff
@@ -259,6 +266,97 @@ testClusters.integTest {
     }
 }
 
+String bwcVersion = "1.13.0.0";
```

Comment:
Lets test with last known latest release, I believe it was 1.13.2 
Ref: https://opendistro.github.io/for-elasticsearch-docs/version-history/

Reply:
For `job-scheduler` and `anomaly-detection`, the latest releases appear to be 1.13.0.0. https://github.com/opendistro-for-elasticsearch/job-scheduler/releases and https://github.com/opendistro-for-elasticsearch/anomaly-detection/releases

Reply:
Got it thank you.